packages/mobile/src/store/auth/reducer.ts

```diff
@@ -1,13 +1,19 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { DataStatus } from '~/common/enums/enums';
-import { signUp } from './actions';
+import { signUp, signIn } from './actions';
 
-type State = {
+export type State = {
   dataStatus: DataStatus;
+  accessToken: string;
+  refreshToken: string;
+  isLoggedIn: boolean;
 };
 
 const initialState: State = {
   dataStatus: DataStatus.IDLE,
+  accessToken: '',
```

Comment:
```AccessToken``` and ```RefreshToken``` should be set and took from mmlk storage, and shouldn't be stored at the storage state.

Reply:
corrected